src/converters.js

```diff
@@ -574,18 +574,25 @@ export function modelChangePostFixer( document ) {
 			return;
 		}
 
+		let sourcePos;
+
+		if ( changes.sourcePosition ) {
+			const howMany = changes.range.end.offset - changes.range.start.offset;
+			sourcePos = changes.sourcePosition._getTransformedByInsertion( changes.range.start, howMany, true );
+		}
```

Comment:
It look like this variable can be used as undefined. Can you move it to `type == 'remove'`  and `type == 'move'`?